.changeset/metal-hornets-travel.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/container-runtime": patch
+---
+---
+"section": "fix"
+---
+Restored old op processing behavior around batched ops to avoid potential regression
```

Comment:
```suggestion
Restore old op processing behavior around batched ops to avoid potential regression
```

Reply:
As discussed - Won't Fix (to match 2.3 release notes)